src/main.cpp

```diff
@@ -2080,6 +2081,47 @@ void PartitionCheck(bool (*initialDownloadCheck)(), CCriticalSection& cs, const
     }
 }
 
+Consensus::VersionBitsConditionChecker checkCSV(Consensus::DEPLOYMENT_CSV);
+Consensus::VersionBitsConditionChecker* const softforkcheck[] = {&checkCSV};
+
+int32_t ComputeBlockVersion(const CBlockIndex* pindexPrev, const Consensus::Params& params)
+{
+    LOCK(cs_main);
+    int32_t nVersion = 0;
+
+    for (unsigned int i = 0; i < sizeof(softforkcheck) / sizeof(softforkcheck[0]); i++) {
+        if (softforkcheck[i]->IsSetFor(pindexPrev, params)) {
+            nVersion |= softforkcheck[i]->Mask(params);
+        }
+    }
+
+    if (nVersion == 0) {
```

Comment:
Why not simply start using the versionbit prefix without voting for any concrete bit instead?